vignettes/howto-dm-df.Rmd

```diff
@@ -215,3 +215,13 @@ flights_dm_all_keys %>%
   dm_examine_constraints() %>%
   tibble::as_tibble()
 ```
+
+## Conclusion {#conclusion}
+
+In this tutorial, we have demonstrated how simple it is to create relational data models from local data frames using {dm}, including setting primary and foreign keys and visualizing the resulting relational model.
+
+## Further reading
+
+`vignette("howto-dm-db")` - This article covers accessing and working with RDBMSs within your R session, including manipulating data, filling in missing relationships between tables, getting data out of the RDBMS and into your model, and deploying your data model to an RDBMS.
```

Comment:
```suggestion
`vignette("howto-dm-db")` -- This article covers accessing and working with RDBMSs within your R session, including manipulating data, filling in missing relationships between tables, getting data out of the RDBMS and into your model, and deploying your data model to an RDBMS.
```

Reply:
Why the additional `-`?

Reply:
Because pkgdown (Pandoc) will transform it to the long hyphen that I think is better suited here

Reply:
![image](https://user-images.githubusercontent.com/8360597/174725549-be4c873c-0bc5-406a-9b31-70cbfc7b86d9.png)

Reply:
(just tested, as I started doubting myself :sweat_smile: )

Reply:
Haha, nice! Didn't know about this feature.

I have converted all of these to long hyphens.